Server/index.js

```diff
@@ -121,6 +121,36 @@ app.post("/cart", (req, res) => {
     .catch((resp) => res.send(resp.error));
 });
 
+
+app.post("/reviews", (req, res) => {
+  const {
+    rating, summary,
+    recommend, body, date, helpfulness, photos,
+  } = req.query;
+  const { reviewerName } = req.query.reviewer_name;
```

Comment:
Why not change line 25 and 31 on NewReview.jsx so that your property names are ratingId and reviewerName? Then you can deconstruct the object in one line here. Alternatively you could pass req.query directly into the following POST params as it already matches the format you need.